website_snippet_carousel_product/templates/snippet.xml

```diff
@@ -0,0 +1,196 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!-- Modify Snippets Menu -->
+    <template
+        id="snippets"
+        inherit_id="website.snippets"
+        name="Snippet Products Carousel Panel Option"
+    >
+        <xpath
+            expr="//div[@id='snippet_feature']//div[hasclass('o_panel_body')]"
+            position="inside"
+        >
+            <t
+                t-snippet="website_snippet_carousel_product.s_product_carousel"
+                t-thumbnail="/website_snippet_carousel_product/static/description/icon.png"
+            />
+        </xpath>
+    </template>
+    <!-- Snippet Carousel Product -->
+    <template id="s_product_carousel" name="Products Carousel">
+        <div class="oe_structure" />
+        <section class="s_product_carousel">
+            <div class="container">
+                <div class="js_product_carousel">
+                    <div class='text-center'>
+                        <i class='fa fa-circle-o-notch fa-spin fa-3x fa-fwg'></i>
+                        <span>Loading Product Carousel</span>
+                    </div>
+                </div>
+            </div>
+        </section>
+    </template>
+    <template id="s_product_carousel_items">
+        <div class="product_carousel_slider oe_website_sale">
+            <input
+                type="hidden"
+                name="product_count"
+                t-if="objects"
+                t-att-value="len(objects)"
+            />
+            <div
+                t-att-id="uniqueId"
+                class="carousel slide d-flex"
+                data-ride="carousel"
+            >
+                <a
+                    class="o_product_carousel_control carousel-control-prev position-relative"
+                    t-att-href="'#' + uniqueId"
+                    role="button"
+                    data-slide="prev"
+                    t-if='num_slides > 1'
+                >
+                    <i class="fa fa-2x fa-chevron-left text-black-50" />
+                    <span class="sr-only">Previous</span>
+                </a>
+                <div class="carousel-inner">
+                    <t t-foreach="objects" t-as="products">
+                        <div
+                            t-attf-class="carousel-item #{'' if products_index else 'active'}"
+                        >
+                            <div class="row">
+                                <t t-foreach="products" t-as="product">
+                                    <div t-attf-class="col-#{int(12 / products_per_slide)}">
```

Comment:
The snippet lost its responsivity:

![Captura de pantalla de 2020-06-11 13-22-36](https://user-images.githubusercontent.com/973709/84384942-b1151c80-abe6-11ea-80f9-ec9617aaf48b.png)

I suggest you to use this instead, or something similar:

```suggestion
                                    <div t-attf-class="col-12 col-sm-6 col-md-#{int(12 / products_per_slide)}">
```